components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -295,6 +309,24 @@ export const PortfolioAsset = () => {
 
   const onNftDetailsLoad = React.useCallback(() => setNftIframeLoaded(true), [])
 
+  const onToggleShowBridgeToAurora = React.useCallback(() => {
+    if (bridgeToAuroraWarningShown) {
+      onOpenRainbowAppClick()
+    } else {
+      setShowBridgeToAuroraModal(prevShoModal => !prevShoModal)
+      localStorage.setItem(bridgeToAuroraWarningShownKey, 'true')
+    }
```

Comment:
This one has a bug where trying to close the modal will open the bridge URL

I recommend creating dedicated open/close functions

Reply:
@josheleonard Good catch . Fixed here 9e6a282b2a0f5906ac5039fb9b311f474f81756f